npm/extensions.bzl

```diff
@@ -11,12 +11,20 @@ load("//npm/private:transitive_closure.bzl", "translate_to_transitive_closure")
 def _extension_impl(module_ctx):
     for mod in module_ctx.modules:
         for attr in mod.tags.npm_translate_lock:
-            lockfile = utils.parse_pnpm_lock(module_ctx.read(attr.pnpm_lock))
-            trans = translate_to_transitive_closure(lockfile, attr.prod, attr.dev, attr.no_optional)
+            # npm_translate_lock MUST run first incase it updates the pnpm-lock.yaml
```

Comment:
Can they be put at the top of the method in their own loop so they all run before any of the other code below?

Reply:
👍